Zeeguu Reader/Pods/SwiftyJSON/README.md

```diff
@@ -0,0 +1,376 @@
+#SwiftyJSON [中文介绍](http://tangplin.github.io/swiftyjson/)
+
+[![Travis CI](https://travis-ci.org/SwiftyJSON/SwiftyJSON.svg?branch=master)](https://travis-ci.org/SwiftyJSON/SwiftyJSON)
+
+SwiftyJSON makes it easy to deal with JSON data in Swift.
```

Comment:
No, cocoapods isn't part of Xcode and will therefore not be run at build time. When you do `pod install`, it will download all dependencies (if needed, won't update them to newer versions, see [`pod install` vs `pod update`](https://guides.cocoapods.org/using/pod-install-vs-update.html)). `pod install` creates an Xcode workspace that contains the project (`Zeeguu Reader` in this case) and a `Pods` project, that contains all pods. That project and the workspace are configured in such a way that every pod and the project are built and put together, but it won't run `pod install` or something.

Now on why that readme is there. I chose to commit the Pods folder (see also the [cocoapods documentation](https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control)). That readme is there, because it is downloaded by cocoapods and put into the Pods folder.